samples/start-app/php-symfony/src/UseCase/NotifyAuthorizationUpdate.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UseCase;
+
+use App\Client\ClientFactoryInterface;
+use GuzzleHttp\Psr7\Response;
+
+final class NotifyAuthorizationUpdate
+{
+    const GET_APP_SCOPES_UPDATE = '/connect/apps/v1/scopes/update?scopes=';
+
+    public function __construct(private readonly ClientFactoryInterface $clientFactory)
+    {
+    }
+
+    public function execute(): Response
+    {
+        // Create instance of client with 'base_uri' configured in .env file and access token stored in database
+        $client = $this->clientFactory->create();
+
+        $apiUrl = self::GET_APP_SCOPES_UPDATE . implode(' ', AppActivation::OAUTH_SCOPES);
```

Comment:
I have an issue with the understanding of this use-case : if you activate with a certain scope and then update with the same, the use-case is then kind of useless, isn't it ?
Shouldn't we add parameters in the app API call, so that we could pass them to the PIM ?

(This comment can be applied to all the apps)

Reply:
see with @gwendalaubert and @tomglvng